QuadradiusR/scripts/game_state.gd

```diff
@@ -27,6 +27,10 @@ func get_current_player() -> String:
 	return self.game_data['current_player_id']
 
 
+func get_other_player_username() -> String:
```

Comment:
Won't this return a structure with both players? Either change the function name or body so it will match.

Reply:
It was a superfluous code